src/main/java/frc/robot/RobotContainer.java

```diff
@@ -144,6 +146,18 @@ public RobotContainer() {
         .getStructArrayTopic("branches", Pose2d.struct)
         .publish()
         .set(REEF_BRANCH_POSES_RED.toArray(new Pose2d[REEF_BRANCH_POSES_RED.size()]));
+
+    NetworkTableInstance.getDefault()
+        .getTable("reef_blue")
+        .getStructArrayTopic("algae", Pose2d.struct)
+        .publish()
+        .set(REEF_BRANCH_POSES_BLUE.toArray(new Pose2d[REEF_ALGAE_POSES_BLUE.size()]));
+
+    NetworkTableInstance.getDefault()
+        .getTable("reef_red")
+        .getStructArrayTopic("algae", Pose2d.struct)
+        .publish()
+        .set(REEF_BRANCH_POSES_RED.toArray(new Pose2d[REEF_ALGAE_POSES_RED.size()]));
```

Comment:
Did you run this and verify the poses in AdvantageScope? I suggest you do that. You'll find a few problems.